.github/docker/README.md

```diff
@@ -0,0 +1,43 @@
+# Content
+
+Dockerfiles and scripts placed in this directory are intended to be used as
+development process vehicles and part of continuous integration process.
+
+Images built out of those recipes may be used with Docker or podman as
+development environment.
```

Comment:
Add a note that below are instructions for Docker users, or sth like that.

Reply:
do you mean I should point somehow this is for docker and not podman? if so, I think it's rather self explanatory 😉 esp. with `podman` being (from CLI perspective) just an alias for `docker`, so all command should work with just a command name change

Reply:
Yes, I know that. But I doubt that someone that don't know hot to build a docker image will know it. Just add a mention that these are docker commands but should work for podman as well or sth like that. Or ignore my comment if you wish, it's not superimportant.

Reply:
Done.